aten/src/ATen/native/cudnn/MHA.cpp

```diff
@@ -286,16 +286,16 @@ auto build_graph_and_tensors(
     // scaled_dot_product_flash_attention_options.set_alibi_mask(true);
   }
 
-  auto seq_q = mha_graph->tensor(fe::graph::Tensor_attributes()
-                                     .set_name("seq_q")
-                                     .set_dim({b, 1, 1, 1})
-                                     .set_stride({1, 1, 1, 1})
-                                     .set_data_type(fe::DataType_t::INT32));
-  auto seq_kv = mha_graph->tensor(fe::graph::Tensor_attributes()
-                                      .set_name("seq_kv")
-                                      .set_dim({b, 1, 1, 1})
-                                      .set_stride({1, 1, 1, 1})
-                                      .set_data_type(fe::DataType_t::INT32));
+  // auto seq_q = mha_graph->tensor(fe::graph::Tensor_attributes()
+  //                                    .set_name("seq_q")
+  //                                    .set_dim({b, 1, 1, 1})
+  //                                    .set_stride({1, 1, 1, 1})
+  //                                    .set_data_type(fe::DataType_t::INT32));
+  // auto seq_kv = mha_graph->tensor(fe::graph::Tensor_attributes()
+  //                                     .set_name("seq_kv")
+  //                                     .set_dim({b, 1, 1, 1})
+  //                                     .set_stride({1, 1, 1, 1})
+  //                                     .set_data_type(fe::DataType_t::INT32));
```

Comment:
What's the difference between the change and the commented out lines?

Reply:
> What's the difference between the change and the commented out lines?

Setting sequence length is not needed as the cudnn engine can figure it out from the shape of the input tensors.

Reply:
Seems that keeping these lines uncommented should be safe.